plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/preferences/PrefPageSQLCompletion.java

```diff
@@ -150,6 +152,7 @@ protected Control createPreferenceContent(@NotNull Composite parent) {
             csInsertSpace = UIUtils.createCheckbox(assistGroup, SQLEditorMessages.pref_page_sql_completion_label_insert_space, null, false, 2);
             csSortAlphabetically = UIUtils.createCheckbox(assistGroup, SQLEditorMessages.pref_page_sql_completion_label_sort_alphabetically, null, false, 2);
             csShowServerHelpTopics = UIUtils.createCheckbox(assistGroup, SQLEditorMessages.pref_page_sql_completion_label_show_server_help_topics, SQLEditorMessages.pref_page_sql_completion_label_show_server_help_topics_tip, false, 2);
+            csShowValues = UIUtils.createCheckbox(assistGroup, SQLEditorMessages.pref_page_sql_completion_label_show_values, SQLEditorMessages.pref_page_sql_completion_label_show_values_tip, false, 2);
```

Comment:
It does not need to be false by default, IMHO.

Reply:
I agree with LonwoLonwo.

Reply:
It's assigned later to the value from the preference store:

```java
csShowValues.setSelection(store.getBoolean(SQLPreferenceConstants.SHOW_VALUES));
```

Reply:
Ok, but wy it has an unchecked status here? What is the reason for this inconsistency?

Reply:
Who cares about its initial value? I just copied the line above.

Reply:
I care. This is one of the places where I can see the default value. I need an extra click to find SQLPreferenceConstants and its default value.

Reply:
Well, the initial state of the GUI control is not supposed to represent the initial value of the preference by any means, so in the end, you will have to find where the preference's default value is initialized.